src/OpenRiaServices.Server.UnitTesting/Test/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM componenets.  If you need to access a type in this assembly from 
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("ad0bd254-e3db-42b8-b10b-32fdf115d319")]
+
+[assembly: AllowPartiallyTrustedCallers]
+[assembly: SecurityRules(SecurityRuleSet.Level1)]
```

Comment:
Is this file copied ? 
I dont think AllowPartiallyTrustedCallers and SecurityRules are standard and can probably be removed

Reply:
removed